org.zdoom.GZDoom.yaml

```diff
@@ -0,0 +1,100 @@
+app-id: org.zdoom.GZDoom
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "18.08"
```

Comment:
Update runtime to 19.08

Reply:
Took some time (As in, I previously looked into this problem three times) but I found out how to update the runtime without the GCC linker crashing.